tools/points-service/main.go

```diff
@@ -474,6 +481,37 @@ func insertOptInWithVault(db *sql.DB, logger *slog.Logger, pubkey, adder, vaultA
 	}
 }
 
+func insertManualValRecord(db *sql.DB,
+	logger *slog.Logger,
+	pubkey,
+	adder string,
+	optedInBlock uint64,
+) error {
+	rwLock.RLock()
```

Comment:
This lock is not needed, `sql.DB` is already handling concurrent access

Reply:
Nice catch, I was pattern matching. Might as well remove the lock altogether then https://github.com/primev/mev-commit/commit/fae763ed7ada7861151c02fc8b24ded8b25eddc3